includes/Model/Story.php

```diff
@@ -230,14 +230,14 @@ public function get_poster_srcset(): string {
 	 * @since 1.0.0
 	 */
 	public function get_title(): string {
-		return (string) $this->title;
+		return $this->title;
```

Comment:
The reason these were put into first place, is that if a developer sets this to false by mistake, ths would give a type error. Is there a worry this might happen? What does it hurt, ensuring these are the correct type?

Reply:
The only way these could be set is through the `load_from_post()` method, where title is always a string. Other fields like the `url` are converted to strings in `load_from_post()`.